lib/VersionHistory/VersionHistoryPane/VersionHistoryPane.js

```diff
@@ -82,13 +85,22 @@ const VersionHistoryPane = ({
     versionsMap,
   ]);
 
-  if (isLoading) return <LoadingPane />;
+  if (isLoading || isLoadingProp) {
+    return (
+      <LoadingPane
+        defaultWidth="20%"
+        dismissible
```

Comment:
It doesn't make sense to me that a loading pane would be dismissible and there is no mention of this change in the ticket, nor an explanation here in the comments. Are you confident that's what you want here?

Reply:
Thank you, Zak. Actually, this ability has been added, in case the user wants to close the version history and return to the original entity (e.g. order) without waiting for the download to complete.

![image](https://user-images.githubusercontent.com/88109087/209967888-9de58fb0-ec46-44b9-837f-b75f80ab19c8.png)